openpathsampling/high_level/move_scheme.py

```diff
@@ -721,6 +722,10 @@ def move_summary(self, steps, movers=None, output=sys.stdout, depth=0):
         depth : integer or None
             depth of submovers to show: if integer, shows that many
             submovers for each move; if None, shows all submovers
+        scheme_copies : lits of :class:`MoveScheme`
```

Comment:
I agree, this is actually a point I wanted to discuss separately. I (still) need it to join data generated from two MoveSchemes. I think I need your insight for a better solution. I will remove it for now to be sure.